.github/workflows/release_packaging_publish.yml

```
@@ -0,0 +1,279 @@
+            artifacts/*.msi
+
+  publish_ecosystems:
+    needs: [evaluate_trigger, package_and_release]
+    if: ${{ needs.package_and_release.result == 'success' && needs.evaluate_trigger.outputs.should_publish_ecosystems == 'true' }}
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Publish to WinGet (optional)
+        if: ${{ vars.WINGET_PACKAGE_IDENTIFIER != '' && (secrets.PERSONAL_ACCESS_TOKEN != '' || secrets.WINGET_TOKEN != '') }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)